style.lua

```diff
@@ -1,8 +1,12 @@
+-- For documentation of Lua tag transformations, see docs/lua.md.
+
+-- Objects with any of the following keys will be treated as polygon
 polygon_keys = { 'building', 'landuse', 'amenity', 'harbour', 'historic', 'leisure', 
       'man_made', 'military', 'natural', 'office', 'place', 'power',
       'public_transport', 'shop', 'sport', 'tourism', 'waterway',
       'wetland', 'water', 'aeroway' }
 
+-- Objects without any of the following keys will be deleted
```

Comment:
Yes, this is true. Objects without any key in this list will be skipped.

Moreover, objects that do have a key in the list, but no column name (for example an object tagged only with capital=yes) will be skipped too.